src/becca/becca_service.ts

```diff
@@ -1,10 +1,10 @@
 "use strict";
 
-const becca = require('./becca');
-const cls = require('../services/cls');
-const log = require('../services/log');
+import becca = require('./becca');
+import cls = require('../services/cls');
+import log = require('../services/log');
```

Comment:
Should these not be `import ... from`?

Reply:
@TriangularCube , i know it looks weird but I wanted to not handle ESM at all. Apparently this `import foo = require` syntax is the TypeScript version of the CJS syntax.

Theoretically we could use ESM interop even if we still use CJS under the hood, but I wanted to approach the conversion one step at a time: first conversion to TypeScript and then to ESM.

We will handle conversion to ESM in #15.

Reply:
Gotcha